File: Ahnhyunwoo/2022-03-25.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>
using namespace std;
vector<int>a,b;
int solution(vector<int> A, vector<int> B) {
    int answer = 0;
    sort(A.begin(), A.end());
    sort(B.begin(), B.end());
    a = A;
    b = B;
    for(int i = 0; i < B.size();i++) {
        int num = lower_bound(A.begin(), A.end(),B[i]) - A.begin();
        if(num - answer > 0) {
            answer++;
        }
    }
    
    return answer;
}

File: juhyunyu/20220325.java
// 프로그래머스
// 문자열 내림차순
import java.util.Arrays;
import java.util.Collections;

class Solution {
    public String solution(String s) {
        String answer = "";
        String[] str = s.split("");
        Arrays.sort(str, Collections.reverseOrder());
        answer = String.join("", str);
        return answer;
    }
}
